src/index.css

```diff
@@ -269,8 +269,8 @@ h3, h4 {
 
 .sticky-outer-wrapper {
   margin: 0;
-  padding-top: calc(var(--x) * 6);
-  margin-top: calc(var(--x) * (-6));
+  padding-top: calc(var(--x) * 8);
+  margin-top: calc(var(--x) * (-8));
```

Comment:
I tested it and it works well. Thanks! However this looks like some kind of magic that should be explained. Could you add a comment of why this is there, how it works, and maybe how it needs to be adjusted when things like headroom change.

Reply:
Basically, it's an invisible trigger to activate sticky header. Since Headroom (when active) decreases the viewport height of a page by disappearing (`positon: fixed;`) after scrolling down, the trigger has to be moved accordingly.

Reply:
And why does it have to be 8 * --x and not 6 * --x?

Reply:
So this is only necessary in for react-stickynode to work alongside headroom? Why would it be needed then with headroom disabled?

Reply:
Not only. It's also necessary because of the flexible height of the sticky header itself and the height calculation of the dashboard, so it all can be snaped into view. This will probably get solved after adopting/completing #130, though.

Reply:
Ok, I adde a comment in the code.